src/esperanto/NewsletterBundle/Resources/config/services.yml

```diff
@@ -1,4 +1,24 @@
+parameters:
+    esperanto_newsletter.event.newsletter.class: esperanto\NewsletterBundle\EventListener\NewsletterBuilderSubscriber
+
 services:
-#    esperanto_newsletter.example:
-#        class: esperanto\NewsletterBundle\Example
-#        arguments: [@service_id, "plain_value", %parameter%]
+    esperanto_newsletter_newsletter_render:
+        class: esperanto\NewsletterBundle\Twig\NewsletterRender
+        arguments:
+            - @service_container
+        tags:
+            - { name: twig.extension }
+
+
+    esperanto_newsletter_subscriber:
+        class: esperanto\NewsletterBundle\Form\Type\SubscriberType
```

Comment:
The name of the class is already defined by the sylius configuration. The parameter called %esperanto_newsletter.form.subscriber.class%